haskell/repl.bzl

```diff
@@ -182,6 +182,7 @@ def _create_HaskellReplCollectInfo(target, ctx):
 
     hs_info = target[HaskellInfo]
 
+    # TODO[GL]: do we need to also take into account narrowed_deps here, everywhere "deps" is checked?
```

Comment:
?

Reply:
I would imagine it is necessary. To rehearse that we would need to add java dependencies to dependencies of Haskell libraries using `modules`.